### ISSUE TITLE
[Bug] Warnings of deprecation and vulnerabilities on step 5 - Deploy

### ISSUE THREAD

**Issue by rffontenelle** [skills/deploy-to-azure#74]
### Summary

In **Step 5: Deploy to a production environment based on labels**, when running the  `deploy-prod.yml` workflow, the **npm install and build webpack** step from **build** job emits several warnings of deprecation and vulnerabilities for the dependencies of the webpack used in the exercise.

### How to reproduce

1. Get to the **Step 5: Deploy to a production environment based on labels**
2. Follow instructions **Activities 1** and then **Activities 2**
3. Once the PR is merged into main, `deploy-prod.yml` workflow will be triggered
4. Browse the logs for this workflow run, check **build** job, **npm install and build webpack** step
5. See warnings in the log

<details><summary>Click here to show the text if the warning shown in the image below</summary>
<pre>
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated source-map-url@0.4.0: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated source-map-resolve@0.5.2: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
npm WARN deprecated buffer@4.9.1: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)

added 1680 packages, and audited 1681 packages in 13s

47 packages are looking for funding
  run `npm fund` for details

12 vulnerabilities (1 low, 3 moderate, 8 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
</pre>
</details> 

![image](https://github.com/skills/deploy-to-azure/assets/1571783/a69b8633-06ff-4c83-97e2-ec26dfce6d29)

### Additional context

Reference of the latest commit from this repository in which I did my exercise: 02e588bddd8c548232dd9d8b96c191f5ae8428d7
